### PR TITLE
[Outlook] (recurrence) Update article

### PR DESCRIPTION
**Pull request by samantharamon**
Updates the recurrence article to address low engagement flags.
- Adds a section to run related Script Lab snippets.
- Updates code samples in article.
- Fixes formatting.

### CODE REVIEW

File: docs/outlook/get-and-set-recurrence.md [hunk 1]
```diff
@@ -1,40 +1,44 @@
...
+ms.date: 05/30/2024
 ms.topic: how-to
 ms.localizationpriority: medium
 ---
 
-# Get and set recurrence
+# Get and set the recurrence of appointments
 
-Sometimes you need to create and update a recurring appointment, such as a weekly status meeting for a team project or a yearly birthday reminder. You can use the Office JavaScript API to manage the recurrence patterns of an appointment series in your add-in.
+Sometimes you need to create and update a recurring appointment, such as a weekly status meeting for a team project or a yearly birthday reminder. Use the Office JavaScript API to easily manage the recurrence patterns of an appointment series in your add-in.
```
Comment (davidchesnut) [inline docs/outlook/get-and-set-recurrence.md:11]:
```suggestion
Sometimes you need to create and update a recurring appointment, such as a weekly status meeting for a team project or a yearly birthday reminder. Use the Office JavaScript API to manage the recurrence patterns of an appointment series in your add-in.
```

File: docs/outlook/get-and-set-recurrence.md [hunk 2]
```diff
@@ -1,40 +1,44 @@
...
 
 > [!NOTE]
 > Support for this feature was introduced in [requirement set 1.7](/javascript/api/requirement-sets/outlook/requirement-set-1.7/outlook-requirement-set-1.7). See [clients and platforms](/javascript/api/requirement-sets/outlook/outlook-api-requirement-sets#requirement-sets-supported-by-exchange-servers-and-outlook-clients) that support this requirement set.
 
-## Available recurrence patterns
+## Review available recurrence patterns
```
Comment (davidchesnut) [inline docs/outlook/get-and-set-recurrence.md:16]: What does "review" mean? Is this material being repeated from another source?

File: docs/outlook/get-and-set-recurrence.md [hunk 3]
```diff
@@ -192,6 +190,19 @@ const recurrenceProperties = recurrence.recurrenceProperties;
 const recurrenceType = recurrence.recurrenceType;
 ```
 
+## Run sample snippets in Script Lab
```
Comment (davidchesnut) [inline docs/outlook/get-and-set-recurrence.md:193]: Nice!

**Review by davidchesnut — APPROVED** [review]
Looks good, just a few suggestions.

File: docs/outlook/get-and-set-recurrence.md (hunk 2 above)
Comment (samantharamon) [reply docs/outlook/get-and-set-recurrence.md:16]: I wanted to make the header consistent with the rest of the headers in the article by starting it with a verb. However, now that I'm reading it again, adding "review" does seem out of place. Changing this to "Recurrence patterns".

Comment (samantharamon) [inline docs/outlook/get-and-set-recurrence.md:16]:
```suggestion
## Recurrence patterns
```

**Pull request merged**